test/react.js

```diff
@@ -26,17 +26,23 @@ test('nested text', function (t) {
 
 test('auto default keys', function (t) {
   var el = mk()
+  // 0
   el.append('p')
-  el.append('p').attr('key', 'testing')
+  // 1
+  var sub = el.append('p').attr('key', 'testing')
+  // 1.0
+  sub.append('p')
+  // 2
   el.append('p').attr('foo', 'bar')
 
-  var tree = el.node().toReact()
+  var tree = el.node().toReact('test')
 
-  t.plan(5)
-  t.equal(tree.key, 'faux-dom-0')
-  t.equal(tree.props.children[0].key, 'faux-dom-0')
+  t.plan(6)
+  t.equal(tree.key, 'test')
+  t.notEqual(tree.props.children[0].key, undefined)
```

Comment:
This test is now pretty vague, all of the `notEqual` ones are. It could be equal to the entire works of Shakespeare and still pass :cry: